src/desktops/dskHostGame.cpp

```diff
@@ -431,6 +431,7 @@ void dskHostGame::UpdatePlayerRow(const unsigned row)
  */
 void dskHostGame::Msg_PaintBefore()
 {
+    Desktop::Msg_PaintBefore();

```

Comment:
Yes, I've been thinking about this. Especially when you change the inheritance to introduce another level this would prove very helpful. However it would be a lot of work.